cypress/integration/use-cases/language-selection.spec.js

```diff
@@ -5,10 +5,35 @@ describe('switching languages', () => {
         cy.get('footer .i18n-widget-language select').select('German (Deutsch)', {
             force: true,
         });
-        cy.contains('Changing the language will redirect you to another domain.');
+        cy.contains('Wenn Du die Sprache wechelst, wirst Du zu einer anderen Domain weitergeleitet.');
         // RegEx taken from https://github.com/datenanfragen/website/pull/564/files#diff-56682af89782eee8f291dc44df0b21449b4203c2bc27990c869d3cd83ba8e036R9
-        cy.contains('Change language')
+        cy.contains('Trotzdem wechseln')
             .should('have.attr', 'href')
             .and('match', /`${Cypress.env('baseUrl_DE') || Cypress.config().baseUrl.replace('1314', '1313')}`/);
     });
 });
+
+describe('language suggestion modal', () => {
+    it('should appear if no cookies set and visiting as DE', () => {
+        cy.visit('/', {
+            onBeforeLoad(win) {
+                Object.defineProperty(win.navigator, 'language', {
+                    value: ['de-DE'],
+                });
+            },
+        });
+        cy.get('#flash-messages').should('contain', 'Diese Seite ist auch in Deiner Sprache verfügbar!');
+    });
+
+    it('should not appear if browser reports english', () => {
+        cy.clearCookies();
+        cy.visit('/', {
+            onBeforeLoad(win) {
+                Object.defineProperty(win.navigator, 'language', {
+                    value: ['en-GB'],
+                });
+            },
+        });
+        cy.get('#flash-messages').should('not.contain', 'Diese Seite ist auch in Deiner Sprache verfügbar!');
+    });
+});
```

Comment:
Why'd you remove the test with the `country` cookie set? That seems important. :D

Reply:
Oops, my bad. Added back in #2fb7811